### ISSUE TITLE
shaders:t="tools" ? Какие есть ещё скомпилированные файлы шейдеров?

### ISSUE THREAD

**Issue by TunderTunder** [GaijinEntertainment/DagorEngine#96]
Потому что я хочу использовать rendinst_semi_trans, или emission или материалы аэродромов из примеров, а консоль говорит такой-то шейдер not found in bin dump.

**Comment by TunderTunder** [comment]
2/3 шейдеров DagorMat2 из MaxTools не работают ни в во viewer ни в игре...

**Comment by NicSavichev** [comment]
Исходники шейдеров находятся тут `X:\DagorEngine.git\prog\daNetGame\shaders
Большая их часть билдятся и используются во вьювере, но, конечно, не все.
Узнать контент бинарника шейдеров проще всего так
в папке `X:\DagorEngine.git\samples\dngSceneViewer\viewer\compiledShaders`
`..\..\..\..\tools\dagor_cdk\windows-x86_64\shaderInfo-dev.exe gameDX12.ps50.shdump.bin `
Вообще, доступные шейдеры (но их может быть больше) описаны тут для 3ds Max плагина
`X:\DagorEngine.git\tools\dagor_cdk\windows-x86_64\plugins-3dsMax\dagorShaders.cfg`

**Comment by TunderTunder** [comment]
Вряд ли эти шейдеры можно использовать в WT CDK?;

**Comment by NicSavichev** [comment]
Да, у ВТ свой набор шейдеров (материалов). Но все они перечислены в конфигах, которые идут к макс-плагину в WT CDK

**Comment by TunderTunder** [comment]
> Да, у ВТ свой набор шейдеров (материалов). Но все они перечислены в конфигах, которые идут к макс-плагину в WT CDK

Но как я уже упоминал некоторые шедейры, особенно специализированные, как rendinst_semi_trans, dynamic_terrain_detail и другие не работают, даже если я повторно экспортирую .dag примеров в /entities.
Можно ли использовать шейдеры из папки X:\SteamLibrary\steamapps\common\War Thunder\compiledShaders?

**Comment by TunderTunder** [comment]
Нет, всё та же мелодия
error during export:
land_mesh_combined_detailed_decal: unknown chan[0] #5 (vbu=0 vbui=0 t=0xa0000), check mesh vertices & vcolor
